docs/websocket.rst

```diff
@@ -0,0 +1,32 @@
+.. _websocket:
+
+=========
+Websocket
+=========
+
+You can enable web sockets if you select ``use_async`` option when creating a project. That indicates whether the project should use web sockets with Uvicorn + Gunicorn.
+
+Usage
+-----
+
+JavaScript example: ::
+
+    > ws = new WebSocket('ws://localhost:8000/') // or 'wss://<mydomain.com>/' in prod
+    WebSocket {url: "ws://localhost:8000/", readyState: 0, bufferedAmount: 0, onopen: null, onerror: null, …}
+    > ws.onmessage = event => console.log(event.data)
+    event => console.log(event.data)
+    > ws.send("ping")
+    undefined
+    pong!
+
+If you are using nGinx instead of Traefik, you should add these lines to nGinx config: ::
+
+    location /websocket/ {
+        proxy_pass http://backend;
+        proxy_http_version 1.1;
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection "upgrade";
+        proxy_read_timeout 86400;
+    }
```

Comment:
I'm not sure about adding this to our docs... I appreciate that Nginx is more widely used, but this is a piece of code that I'd rather NOT maintain and instead link to Nginx, which is more likely to remain updated.

Can we rephrase this to something more generic? Something along the lines of 

> "if you don't use Traefik, you might have to configure your reverse proxy accordingly (example [with Nginx](https://www.nginx.com/blog/websocket-nginx/))"